tests/Parse/ParseSchemaTest.php

```diff
@@ -411,7 +429,7 @@ public function testBadSchemaGet()
      */
     public function testBadSchemaSave()
     {
-        $this->setExpectedException('\Parse\ParseException');
+        $this->setExpectedException('\Exception');
```

Comment:
One last thing. Is there a particular reason you changed the expected exception here?

Reply:
https://github.com/dplewis/parse-php-sdk/blob/schema-index/src/Parse/ParseSchema.php#L277

It throws exception instead of parse exception. The test were failing locally for some reason.

On a side note testBadSchemaDelete doesn't pass locally for me. If I remove the spaces from `$badClassName = "<Bad~ Class~ Name>` it passes.

Reply:
Ahh that's something I missed. That change is good then. And quick question, are you running these tests on a windows machine by chance?

Reply:
MacOS 10.12.6 Sierra

Reply:
Hmm, good on mine and we're on the same setup. There's certainly a reason for it but I wonder what 🤔 . Anyways this is all good then.

Reply:
Thanks for waiting. Now onto 1.4.0 👍